Docker.DotNet.Tests/Docker.DotNet.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
I was about to do that, but I first wanted you guys to easily review it so the diff here is easier without a bunch of delete/move that GH will do. 

Once you give the OK on this PR, I'll reset and push again with the src/ and test/ folders in place.